### PR TITLE
Add some sites to the global dark list

### PR DESCRIPTION
**Pull request by sauce420**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -62,6 +62,7 @@ avengedsevenfold.com
 azeria-labs.com
 azm.azerialabs.com
 badboybill.com
+based.cooking
```
Comment (Gusted) [inline src/config/dark-sites.config:65]: image

Doesn't seem that dark to me.

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -835,6 +838,7 @@ www.directvnow.com
 www.gotimelinr.com
 www.netflix.com
 www.razer.com
+www.stargazing.net
```
Comment (Gusted) [inline src/config/dark-sites.config:841]: You can remove this entry.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 1 above)
Comment (sauce420) [reply src/config/dark-sites.config]: Could've sworn there was a dark theme at one time [This issue on the website's repo says so](https://github.com/LukeSmithxyz/based.cooking/issues/283) but it doesn't matter

**Comment by sauce420** [comment]
I'll make a new PR with suggested changes

**Review by Gusted — APPROVED** [review]
👍🏽

**Comment by sauce420** [comment]
My first successful PR! Thanks for the quick response btw

**Pull request merged**